samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/LayerLists.razor

```diff
@@ -24,6 +27,26 @@
             </WebScene>
             <LayerListWidget Position="OverlayPosition.TopRight" OnListItemCreatedHandler="OnListItemCreated" />
         </SceneView>
+        <div>
```

Comment:
I think this whole thing should be it's own page. I'm happy to know that multiple views work on a page, but it gets cluttered for quick viewing/consumption. You've also got this all inside the `_windowWidth` check, so it will disappear on mobile layouts.

Reply:
corrected